README.md

```diff
@@ -1,273 +1,132 @@
-};
-// Upsert the data into your index
-await index.upsert(record);
-```
-
-### Querying
-
-#### Querying with vector values
-
-The query method accepts a large number of options. The dimension of the query vector must match the dimension of your index.
-
-```typescript
-type QueryOptions = {
-  vector: number[];
-  topK: number;
-  includeVectors?: boolean;
-  includeMetadata?: boolean;
-};
-```
-
-For example, to query by vector values you would pass the `vector` param in the options configuration. For brevity sake this example query vector is tiny (dimension 2), but in a more realistic use case this query vector would be an embedding outputted by a model. Look at the [Example code](#example-code) to see more realistic examples of how to use `query`.
-
-```typescript
-> await index.query({ topK: 3, vector: [ 0.22, 0.66 ]})
-{
-  matches: [
-    {
-      id: '6345',
-      score: 1.00000012,
-      vector: [],
-      metadata: undefined
-    },
-    {
-      id: '1233',
-      score: 1.00000012,
-      vector: [],
-      metadata: undefined
-    },
-    {
-      id: '4142',
-      score: 1.00000012,
-      vector: [],
-      metadata: undefined
-    }
-  ],
-  namespace: ''
-}
-```
-
-You include options to `includeMetadata: true` or `includeVectors: true` if you need this information. By default these are not returned to keep the response payload small.
-
-### Update a record
-
-You may want to update vector `vector` or `metadata`. Specify the id and the attribute value you want to update.
-
-```typescript
+//Update Data
 await index.upsert({
-  id: "18593",
-  metadata: { genre: "romance" },
+  id: "upstash-rocks",
+  metadata: {
+    title: 'Star Wars',
+    genre: 'sci-fi',
+    category: 'classic'
+  }
 });
-```
 
-### Fetch records by their IDs
-
-```typescript
-const fetchResult = await index.fetch(["id-1", "id-2"]);
-```
+//Delete record
+await index.delete("upstash-rocks");
 
-### Delete records
+//Delete many by id
+await index.delete(["id-1", "id-2", "id-3"]);
 
-For convenience there are several delete-related options. You can verify the results of a delete operation by trying to `fetch()` a record.
+//Fetch records by their IDs
+await index.fetch(["id-1", "id-2"]);
 
-#### Delete one
+//Fetch records with range
+await index.range({
+      cursor: 0,
+      limit: 5,
+      includeVectors: true,
+});
 
-```typescript
-await index.delete("id-to-delete");
-```
+//Reset index
+await index.reset();
 
-#### Delete many by id
+//Info about index
+await index.info();
 
-```typescript
-await index.delete(["id-1", "id-2", "id-3"]);
+//Random vector based on stored vectors
+await index.random();
 ```
 
-### Info
+## Troubleshooting
 
-To get information about your index, you can use the client like so:
+We have a
+[dedicated page](https://docs.upstash.com/vector/sdks/javascriptsdk/troubleshooting) for common problems. If you can't find a solution, please [open an issue](https://github.com/upstash/vector-js/issues/new).
```

Comment:
This page does not exist

Reply:
Yeah, I assumed there will be a page soon enough 😄